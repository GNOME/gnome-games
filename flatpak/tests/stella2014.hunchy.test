[Retro Reftest]
Path=/stella2014/hunchy
Core=/app/lib/libretro/stella2014_libretro.so
Medias=hunchy.a26;

[Options]

[Frame 1]
Video=stella2014.hunchy.1.png
