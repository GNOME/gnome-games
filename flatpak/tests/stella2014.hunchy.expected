/stella2014/hunchy/Boot: OK
/stella2014/hunchy/Options: OK
/stella2014/hunchy/1/FastForward: OK
/stella2014/hunchy/1/Run: OK
/stella2014/hunchy/1/Video: OK
